src/akkudoktoreos/visualize.py

```diff
@@ -81,13 +83,14 @@ def visualisiere_ergebnisse(
         plt.grid(True)
 
         # Temperature forecast
-        plt.subplot(3, 2, 5)
-        plt.title("Temperature Forecast (°C)")
-        plt.plot(hours, temperature, label="Temperature (°C)", marker="x")
-        plt.xlabel("Hour of the Day")
-        plt.ylabel("°C")
-        plt.legend()
-        plt.grid(True)
+        if temperature is not None:
```

Comment:
genetic.py line 26: temperature is not optional. so it can't be empty, or is there a chance?

Reply:
it's unused besides the display at optimization, therefore optional